packages/ssi-types/src/mapper/credential-mapper.ts

```diff
@@ -217,13 +227,19 @@ export class CredentialMapper {
       isJwtEncoded || isJwtDecoded
         ? CredentialMapper.jwtDecodedPresentationToUniformPresentation(decoded as JwtDecodedVerifiablePresentation, false)
         : (decoded as IVerifiablePresentation)
+
+    // At time of writing Velocity Networks does not conform to specification. Adding bare minimum @context section to stop parsers from crashing and whatnot
+    if (!uniformPresentation['@context']) {
```

Comment:
Please make this an option. I don't want us to blindly do this, as this is non-spec compliant

Reply:
Added optional repairMissingVpContext option